testsuite/lb_gpu_leesedwards_gridoffset.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
please combine all the tests into one testclass

Reply:
How can I reset the LB to initial state?

Reply:
You can't, as LB can not be disabled.

Reply:
There is an open PR #1574 which should be merged and then used @fweik